lisa/wa_results_collector.py

```diff
@@ -163,11 +163,16 @@ def __init__(self, base_dir=None, wa_dirs=".*", plat_info=None,
 
         kernel_refs = {}
         for sha1 in df['kernel_sha1'].unique():
+            if sha1 is None:
+                continue
             if kernel_repo_path:
                 try:
                     symref = find_shortest_symref(kernel_repo_path, sha1)
                 except ValueError:
-                    symref = sha1
+                    try:
+                        symref = get_commit_message(kernel_repo_path, sha1)
+                    except:
```

Comment:
`except` needs an exception type to catch, otherwise it will catch things that you definitely don't want to catch: https://docs.python.org/3/library/exceptions.html#exception-hierarchy

I assume that you are mostly interested in catching `subprocess.CalledProcessError`

Reply:
Will do.